### DISCUSSION TITLE
Any way to install this package and use it beside V1 in the same project?

### DISCUSSION THREAD

**Discussion by Bewinxed** [pydantic/pydantic#5668] [Question] [answered]
I have a project where i use pydantic extensively, however, i tried to install V2 to migrate my models as a test but i found out that other libraries that depend on pydantic (such as fastapi) immediately broke.

Any way to install this as a separate package and use it incrementally until the world catches up?

Thanks!

**Comment by Kludex** [comment, marked as answer]
We are going to release a Pydantic with a different namespace (`pydantic-next`), hopefully tomorrow.

- Related to https://github.com/pydantic/pydantic/discussions/4649

**Comment by Bewinxed** [comment]
Godbless, Ty ! <3
